src/item_pocket.cpp

```diff
@@ -852,7 +852,7 @@ void item_pocket::general_info( std::vector<iteminfo> &info, int pocket_number,
         info.back().bNewLine = true;
         info.push_back( iteminfo( "BASE", _( "Maximum item length: " ),
                                   string_format( "<num> %s", length_units( data->max_item_length ) ),
-                                  iteminfo::lower_is_better,
+                                  iteminfo::no_flags,
```

Comment:
This is the maximum item length for items pockets can contain.
Bigger is better in that case ... since it allows you to put a wider selection of items into the pocket.

Reply:
looks like I was more turned around than I thought!